src/Symfony/Component/HttpKernel/KernelInterface.php

```diff
@@ -20,6 +20,8 @@
  *
  * It manages an environment made of application kernel and bundles.
  *
+ * @method string getBuildDir() Returns the build directory - not implementing it is deprecated since Symfony 5.2.
```

Comment:
I think it's `@method getBuildDir()`, no?

Reply:
I would also add more information about when to use `getBuildDir()` (If you could update the description of `getCacheDir()` as well to better describe when to use it, that would be wonderful).